target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -158,6 +158,14 @@ netgear,ex6130)
 	ucidef_set_led_netdev "wlan5g" "ROUTER (green)" "$boardname:green:router" "wlan0"
 	ucidef_set_led_netdev "wlan2g" "DEVICE (green)" "$boardname:green:device" "wlan1"
 	;;
+netgear,ex6100-v1)
+	ucidef_set_rssimon "wwan" "200000" "1"
+	ucidef_set_led_rssi "rssirouterred" "RSSI ROUTER RED" "$boardname:red:router" "wwan" "1" "66"
+	ucidef_set_led_rssi "rssiroutergreen" "RSSI ROUTER GREEN" "$boardname:green:router" "wwan" "34" "100"
+	ucidef_set_rssimon "wlan0" "200000" "1"
+	ucidef_set_led_rssi "rssiclientred" "RSSI CLIENT RED" "$boardname:red:client" "wlan0" "1" "66"
+	ucidef_set_led_rssi "rssiclientgreen" "RSSI CLIENT GREEN" "$boardname:green:client" "wlan0" "34" "100"
```

Comment:
I don't think there is "wwan" in OpenWrt by default?
Congrats for how you reproduce the LED behavior though.
I'd keep the client part here (as it will out of the box). Just make sure you adjust the names if you follow my guidelines from above, e.g.
```
ucidef_set_led_rssi "clientlinkred" "Client Poor Link" "$boardname:red:client-link" "wlan0" "1" "66"
ucidef_set_led_rssi "clientlinkgreen" "Client Best Link" "$boardname:green:client-link" "wlan0" "34" "100"
```

Note that my names are meant as suggestions (based on the manual), I'm open for other ideas.

Reply:
Because that LED is designed to indicate the connection with upstream router.

Reply:
![image](https://user-images.githubusercontent.com/12028138/78578176-b611c480-7861-11ea-9900-85c092ce1b18.png)
http://www.downloads.netgear.com/files/GDC/EX6100/EX6100_UM_27Oct2015.pdf

Reply:
Can `rssileds` update LEDs based on two wireless interface?

Reply:
I understood what the LED is meant to do, however, in default OpenWrt configuration there is no "wwan", and thus this config won't make sense. In addition, rssileds package does not like being configured for an interface that doesn't exists and will cause issues with high load etc.

> Can rssileds update LEDs based on two wireless interface?

I don't know, I've never tried that. However, in any case this will only work for two _existing_ interfaces, so it still is no option for default config (unless wwan is present in buildbot snapshots and I'm just not aware).

Reply:
Changed `wwan` to `wlan1`.

Reply:
First: is it working? And do you still have "normal" load? Just curious...
Second: No. These LEDs are meant for connection to the AP, and not as a second client link LED. Despite, you told me that 5 GHz is not working anyway. So, since you have only one working Wiki anyway, just use that for the client link and fine.